qiime2/sdk/usage.py

```diff
@@ -6,16 +6,38 @@
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
 
+"""
+The Usage module enables generating examples for multiple QIIME 2 interfaces.
+
+The Usage API provides an interface-agnostic way for QIIME 2 plugin developers
+to define examples of how to use their plugin’s actions. This enables the
+programmatic generation of examples for all QIIME 2 interfaces, eliminating the
+need to maintain specific examples for multiple interfaces.
+
+Examples
+--------
+Please see `qiime2.core.testing.examples` for examples.
```

Comment:
Putting a note here, but please update _everywhere_ in this document:

These docstrings are restructuredtext, not markdown, so you'll need to tweak the backticks to work as expected: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
I've tried to follow the numpydoc style guide.  Please see the note [here]( https://numpydoc.readthedocs.io/en/latest/format.html#common-rest-concepts):

> Variable, module, function, and class names should be written between single back-ticks (\`numpy\`).

Reply:
Fair enough, but have you looked at the rendered dev docs? The single-backticks don't render correctly.

![Screen Shot 2020-11-09 at 3 44 15 PM](https://user-images.githubusercontent.com/274668/98605343-77823380-22a2-11eb-8c6d-82fe15c691f7.png)

You'll notice that other modules in qiime2 also use the double-backtick syntax, so I don't think you'll be able to change a config option on the dev-docs (at least not without breaking everything else). For an example, take a look at [`qiime2.metadata`](https://github.com/qiime2/qiime2/tree/master/qiime2/metadata).